ts/src/pro/poloniex.ts

```diff
@@ -201,8 +192,7 @@ export default class poloniex extends poloniexRest {
          * @returns {object} a [ticker structure]{@link https://docs.ccxt.com/en/latest/manual.html#ticker-structure}
          */
         await this.loadMarkets ();
-        const name = 'ticker';
-        return await this.subscribe (name, name, false, [ symbol ], params);
+        return await this.watchTickers ([ symbol ], params);
```

Comment:
@pcriadoperez Also we can't do this, because `watchTickers` returns a dictionary indexed by a symbol